### ISSUE TITLE
How to use the clearDoublettes method?

### ISSUE THREAD

**Issue by solitud** [dereuromark/cakephp-queue#114]
I don't quite understand how to call the `clearDoublettes` method.
I want to build some kind of notification mailer for new entries in a forum.
When users send the same message to the board several times by mistake, other users shall only be notified once.

Would be great if someone could point me in the right direction.
I am using the 2.x branch for CakePHP 2.4

**Comment by dereuromark** [comment]
I am not so sure this should be implemented in the queue itself.
the queue only delivers ready-to-be sent things.
If you want to avoid mult-send you could set a flag in the DB for such a user somewhere to avoid multi-sending in a certain timeframe or for multiple items created in the same instance etc.

If you really want to aggregrate, that should also be done separaretly probably.

What you could do:
Ask if a job with a certain "type" and details is already in the queue, and if so, skip.
But again, not sure if that really works well.

**Comment by solitud** [comment]
I see.
Best thing would be to prevent that duplicates will be added to the queue in the first place.
> What you could do:
Ask if a job with a certain "type" and details is already in the queue, and if so, skip.

*Type* is *group* and *details* is *data* I guess? Just to be sure: there is no existing method that checks for possible doublettes? I just wondered because I saw that `clearDoublettes` method and thought that maybe was implemented for that purpose. But I guess it should not be too hard to do that on my own..
Thanks a lot!
